src/magmaviz/boxplot.py

```diff
@@ -20,5 +24,52 @@ def boxplot(df, x, y, facet=False):
     Examples
     --------
     >>> from magmaviz.magmaviz import boxplot
-    >>> boxplot(mtcars, horsepower, cars, facet=True)
+    >>> boxplot(cars, "Miles_per_Gallon", "Origin", facet=True)
```

Comment:
y is of type list. Should the input be in []? Like ["Origin"]? You should probably test how it works in case a user wants to pass multiple categorical columns and leave an example here for those cases.